bundles/org.openhab.binding.buienradar/src/main/java/org/openhab/binding/buienradar/internal/buienradarapi/BuienradarPredictionAPI.java

```diff
@@ -143,7 +143,7 @@ public ZonedDateTime getActualDateTime() {
         try {
             result = HttpUtil.executeUrl("GET", address, TIMEOUT_MS);
         } catch (IOException e) {
-            logger.warn("IO Exception when trying to retrieve Buienradar results", e);
+            logger.warn("IO Exception when trying to retrieve Buienradar results");
```

Comment:
Let's make this a debug log. warning logging is intended for more serious errors anyway. Maybe do add the specific io message to the log.
```suggestion
            logger.debug("IO Exception when trying to retrieve Buienradar results: {}", e.getMessage());
```

Reply:
Maybe the even better solution